doc/releases/migration-guide-3.7.rst

```diff
@@ -523,6 +523,11 @@ Display
         };
     };
 
+* The ``orientation-flipped`` property has been removed from the SSD16XX
```

Comment:
Thanks @danieldegrasse . But please add a reference the PR which did this change
``` (:github:`123456`) ```.

Reply:
Done, thanks